MatrixSDK/Crypto/MXCrypto.h

```diff
@@ -235,18 +218,24 @@ FOUNDATION_EXPORT NSString *const kMXCryptoRoomKeyRequestCancellationNotificatio
                complete:(void (^)(void))complete;
 
 /**
- Download the device keys for a list of users and stores them into the crypto store.
+ Get the device keys for a list of users.
+
+ Keys will be downloaded from the matrix homeserver and stored into the crypto store
```

Comment:
This happens only when forceDownload is YES?

Reply:
no. It also happens "if the information in the store is not up-to-date."